backend/src-common/src/main/java/org/eclipse/sw360/mail/MailUtil.java

```diff
@@ -98,6 +99,7 @@ private void setBasicProperties() {
         enableDebug = loadedProperties.getProperty("MailUtil_enableDebug", "false");
         supportMailAddress = loadedProperties.getProperty("MailUtil_supportMailAddress","");
         smtpSSLProtocol = loadedProperties.getProperty("MailUtil_smtpSSLProtocol", "");
+        smtpSSLTrust = loadedProperties.getProperty("MailUtil_smtpSSLTrust", "");
```

Comment:
please make MailUtil_smtpSSLTrust=* by default to allow all the host as trusted,

Reply:
Done.